chaco/plot_containers.py

```diff
@@ -321,51 +140,59 @@ def _do_layout(self):
         else:
             align = "max"
 
-        return self._do_stack_layout(components, align)
+        return stack_layout(self, components=components, align=align)
```

Comment:
there are bugs in `HStackedContainer` and the `HStackedContainer._do_layout` - if those get fixed, we will be able to remove this `HPlotContainer._do_layout` method and `HPlotContainer.valign` trait.

Reply:
https://github.com/enthought/enable/pull/780